.github/workflows/automatic-updates.yml

```
@@ -27,8 +27,8 @@ jobs:
           token: ${{ secrets.GITHUB_TOKEN }}
           branch: update-branch
           base: main
-          commit-message: "Update to ${{ steps.updt.outputs.updated-versions }}"
-          title: "Update to ${{ steps.updt.outputs.updated-versions }}"
+          commit-message: "Update to ${{ steps.updt.outputs.result.updatedVersions.join(', ') }}"
+          title: "Update to ${{ steps.updt.outputs.result.updatedVersions.join(', ') }}"
```

Since the title is used for the downstream PR
```suggestion
          title: "feat: Node.js ${{ steps.updt.outputs.result.updatedVersions.join(', ') }}"
```